processed 2 tasks

task 1 'publish'. lines 5-24:
Error: Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: ModuleVerificationFailure, source: None } }
